content/docs/use-cases/sharing-resources-efficiently.md

```diff
@@ -0,0 +1,96 @@
+# Sharing Resources Efficiently
+
+Data science teams need to handle large files, rotate the use of special
+processors, and minimize data transfers. This involves provisioning and managing
+resources such as massive on-prem or cloud data stores and powerful servers,
+which is expensive and time consuming.
+
+![](/img/resource-pool.png) _Fragile and disjointed pool of limited resources_
+
+Minimizing operational cost and complexity can make the difference, for example,
+when:
+
+- Multiple users work on the same shared server.
+- There's a centralized data storage unit or single-entry cluster.
+- There's a single computing environment with access to the data needed to
+  reproduce full [experiments](/doc/user-guide/experiment-management).
+- Distributing GPU time among people or processes for training machine learning
+  models
+
+In DVC, a built-in <abbr>caching</abbr> mechanism already provides individual
+<abbr>projects</abbr> with basic
+[dataset optimization](/doc/user-guide/large-dataset-optimization). It
+de-duplicates file contents automatically. And by linking files from cache to
+<abbr>workspace</abbr>, it achieves near-instantaneous switching between
+[versions of data](/doc/use-cases/versioning-data-and-model-files), results,
+etc.
+
+These benefits can extend to multiple DVC repository clones, or even different
+projects altogether. Just set up
+[the same cache](/doc/user-guide/how-to/share-a-dvc-cache) directory in all of
+them so that they share a central data location, optimizing local (or network)
+storage.
+
+![](/img/shared-server.png) _One data store shared among people or projects_
+
+Additionally, [remote storage](/doc/command-reference/remote) (Amazon S3, SSH,
+Google Drive, etc.) can also be shared by multiple projects to synchronize their
+data caches (see `dvc push` and `dvc pull`). This can reduce service costs by
+consolidating the backups.
```

Comment:
backups - not the best term here. To some extent we are interested more in these optimizations when people store datasets there ... even for models - this is not about backups to my mind - it's a primary storage

Reply:
though with shared cache scenario, it can be a backup when we specifically use the external cache that has everything in it as a primary storage, but it's not clear from the doc, and it is one of the possible scenarios

Reply:
Clarified (I think).